### ISSUE TITLE
The drop down menus (display as, primitives) should take minimum-possible width of the Add-on's GUI

### ISSUE THREAD

**Issue by raindropsfromsky** [jayanam/fast-carve#40]
The drop-down menus take up too much width of the Fast-Carve GUI. As a result, the labels are not visible until the GUI is widened to a very large extent. By that time the labels become fully visible, the Fast-Carve GUI takes up too much of the screen width.

For example, as shown below, the "Display As:" label is partially hidden, although the GUI is already 4.5 cm wide.
![image](https://user-images.githubusercontent.com/9047168/62668937-5352eb00-b9ab-11e9-90dc-90d8e6b6a506.png)

As shown below, the label is revealed only when the GUI is widened to 6 cm.
![image](https://user-images.githubusercontent.com/9047168/62669075-e55af380-b9ab-11e9-831f-8617c5d73d5f.png)

The same thing happens with the "primitives" drop-down menu also.

**Desired:** All labels should become visible even when the GUI is bare-minimum wide.

In other words, the drop down menus (display as, primitives) should take up minimum width of the GUI.
